### PR TITLE
Create main.pm for CASP

### PR DESCRIPTION
**Pull request by kravciak**
At first it will share needles with SLES as YaST installer is based on SP2. When it differs too much we can create separate git repository

Needles: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/247

Local runs:
VMX: http://dhcp91.suse.cz/tests/3011
DVD: http://dhcp91.suse.cz/tests/3014

### CODE REVIEW

File: lib/utils.pm [hunk 1]
```diff
@@ -239,6 +240,10 @@ sub is_jeos() {
     return get_var('FLAVOR', '') =~ /^JeOS/;
 }
 
+sub is_casp() {
```
Comment (okurz) [inline lib/utils.pm]: as long as `is_casp` is only used in the sle main.pm I recommend to also define the sub there (trying to always define the code to the nearest point where it is needed while avoiding duplication).

File: products/casp/main.pm [hunk 2]
```diff
@@ -0,0 +1,82 @@
+use strict;
+use warnings;
+use testapi qw/check_var get_var set_var/;
+use needle;
+#use lockapi;
+#use File::Find;
```
Comment (okurz) [inline products/casp/main.pm]: just delete disabled code

File: products/casp/main.pm [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    }
+    loadtest "installation/addon_products_sle.pm";
+    loadtest "installation/system_role.pm";
+    loadtest "installation/partitioning.pm";
+    loadtest "installation/partitioning_finish.pm";
+    loadtest "installation/releasenotes.pm";
+    loadtest "installation/installer_timezone.pm";
+    if (get_var('IMPORT_USER_DATA')) {
+        loadtest 'installation/user_import.pm';
+    }
```
Comment (okurz) [inline products/casp/main.pm]: AFAIU 'import_user_data' should never make sense on microos

Comment (kravciak) [inline products/casp/main.pm]: Workflow was specified yesterday in fate#321754 - I will update

File: lib/utils.pm (hunk 1 above)
Comment (kravciak) [inline lib/utils.pm]: I expect to use it also in bootloader and maybe during installation, but I can move it to main.pm for now

**Comment by kravciak** [comment]
Please check required needles https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/247

**Pull request merged**